src/measure_theory/measure_space.lean

```diff
@@ -997,6 +1001,17 @@ begin
     outer_measure.restrict_apply]
 end
 
+/-- If `s` is a measurable set, then the measure of `t` restricted to `s` equals the measure of
+`t ∩ s`. This is an alternate version of `measure.restrict_apply`, requiring that `s` is measurable
+instead of `t`. -/
```

Comment:
```suggestion
/-- If `s` is a measurable set, then the outer measure of `t` with respect to the restriction of
the measure to `s` equals the outer measure of `t ∩ s`. This is an alternate version of 
`measure.restrict_apply`, requiring that `s` is measurable instead of `t`. -/
```